.travis.yml

```diff
@@ -54,6 +54,8 @@ jobs:
         -   name: lint
             stage: lint
             script: flake8
+        -   name: Type annotation checking
+            script: poetry run mypy hwilib/base58.py
```

Comment:
Could you add an `install` phase that just installs `mypy` and whatever else is needed to run the check?

Reply:
Updated to use an `install` phase, and dropped the `mypy` dev dependency.